lighthouse-treemap/types/treemap.d.ts

```diff
@@ -0,0 +1,28 @@
+import '../../types/treemap';
```

Comment:
I don't see any errors in vscode now, but `yarn tsc -p lighthouse-treemap/` fails spectacularly. @brendankenny I'm quite lost here, do you have any advice?

Reply:
I think `typeRoots` might be your problem